src/pages/content-edit/form.js

```diff
@@ -0,0 +1,63 @@
+import m from "mithril";
+
+import capitalize from "lodash.capitalize";
+import name from "./name.js";
+import * as children from "../../types/children.js";
+
+import css from "./form.css";
+
+export function view(ctrl_unused, options) {
```

Comment:
Okay. I was just trying to make it clear that this controller isn't actually used for anything.